packages/wallet-standard/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@glow-xyz/wallet-standard",
+    "version": "0.1.0-alpha.0",
+    "type": "module",
+    "sideEffects": false,
+    "main": "./lib/cjs/index.js",
+    "module": "./lib/esm/index.js",
+    "types": "./lib/types/index.d.ts",
+    "exports": {
+        "require": "./lib/cjs/index.js",
+        "import": "./lib/esm/index.js",
+        "types": "./lib/types/index.d.ts"
+    },
+    "files": [
+        "lib",
+        "src"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "build": "pnpm run clean && pnpm run tsc && pnpm run package",
+        "clean": "shx mkdir -p lib && shx rm -rf lib",
+        "tsc": "tsc --build --verbose tsconfig.json",
+        "package": "shx echo '{ \"type\": \"commonjs\" }' > lib/cjs/package.json",
+        "fmt": "prettier --write '{*,**/*}.{ts,tsx,js,jsx,json}'",
+        "lint": "prettier --check '{*,**/*}.{ts,tsx,js,jsx,json}' && eslint .",
+        "lint:fix": "pnpm run fmt && eslint --fix ."
+    },
+    "dependencies": {
+        "@glow-xyz/glow-client": "^1.3.0",
+        "@wallet-standard/features": "^0.1.0-alpha.4",
+        "@wallet-standard/solana-features": "^0.1.0-alpha.3",
+        "@wallet-standard/standard": "^0.1.0-alpha.5",
```

Comment:
These could be redeclared as dev dependencies, since they are only imported as types.